cmd/agent/daemon/enrichment/redactor.go

```diff
@@ -1,59 +0,0 @@
-package enrichment
-
-import (
-	"context"
-	"regexp"
-	"strings"
-
-	castpb "github.com/castai/kvisor/api/v1/runtime"
-)
-
-type SensitiveValueRedactor struct {
```

Comment:
Dropped SensitiveValueRedactor as it requires to fix tests and update output signatures. We did not set any values and it was not used. We process the same logic in backend already. Having duplicate logic here seems like a cpu waste and could be useful only if at some point users need to export data to their siem.